packages/framework/client-logger/external-telemetry/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { IExternalTelemetry, ExternalTelemetryEventName } from "./common";
+export {
+	IContainerTelemetry,
+	ContainerTelemetryEventName,
+	ContainerTelemetryEventNames,
+	ContainerConnectedTelemetry,
+	ContainerClosedTelemetry,
+	ContainerDisconnectedTelemetry,
+	ContainerAttachingTelemetry,
+	ContainerAttachedTelemetry,
+} from "./container";
```

Comment:
Are there specific reasons to export all of these? As long as this package is intended as an example/reference implementation more than something extendable, I think all consumers need to use this are the function and the config type below.

Reply:
Was talking with Josh and he mentioned that if we don't export the telemetry types then tsdoc won't be generated for them so I opted to leave them exported.